pacsman/integration_test.py

```diff
@@ -90,7 +94,7 @@ def test_local_series_for_study(local_client):
     assert len(series_datasets) > 1
     assert series_datasets[0]
     for ds in series_datasets:
-        assert ds.NumberOfImagesInSeries >= 1
+        assert ds.NumberOfSeriesRelatedInstances >= 1
         assert ds.InstitutionName
 
```

Comment:
This will test the new `fetch_image_as_file` method, I reran the tests on my machine and they pass.
```
 @pytest.mark.integration
 @pytest.mark.local
 def test_local_fetch_image(local_client, tmpdir):
     # Patient ID E3148
     series_id = '1.2.392.200193.3.1626980217.161129.153348.41538611151089740341'
     # first image in series
     instance_id = '1.2.392.200193.4.1626980217.161129.153348.13597379231089740343'
     local_client.dicom_dir = tmpdir

     image_path = local_client.fetch_image_as_file(series_id, instance_id)
     assert image_path
     assert len(os.listdir(tmpdir)) == 1
```